apps/expo/src/app/workout/startExisting/index.tsx

```diff
@@ -18,18 +18,11 @@ export default function createNew() {
 
```

Comment:
there is a lot of questionable code in this file. :( 
we also run into the same problem of 

```
 ERROR  Warning: Each child in a list should have a unique "key" prop.
 ```

Reply:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/5ce9182b-e688-4b39-a1e3-2795ee2d933f)

why there is 2 workout list,  

```
<View style={{height: 1, backgroundColor: '#CACACA', marginHorizontal: 13, opacity: 0.3, top: 10, }}/>
```

that view doesn't do anything.